### DISCUSSION TITLE
也许我们也可以整一个部署脚本？

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#726] [Ideas]
究极保姆，主要是添加 cookie 会有人不能理解 yaml 的格式，但我觉得我们的项目部署已经挺容易的了

隔壁的

https://github.com/BennettChina/adachi-bot-installer

**Comment by Arondight** [comment]
我感觉没必要，人都是有学习能力的，而且自述文件里写的也挺明白了，况且我们的初衷都是自用，在这个基础上已经极大地照顾大家了，没必要更进一步
